### DISCUSSION TITLE
这个如何部署到本地服务器上面啊

### DISCUSSION THREAD

**Discussion by WG-itgm** [smart-test-ti/SoloX#152] [Q&A]
这个如何部署到本地服务器上面啊

**Comment by rafa0128** [comment]
![image](https://user-images.githubusercontent.com/24454096/236109844-97c450a3-3009-4828-b9f4-153fac93b03f.png)

**Reply by WG-itgm** [reply]
3q

**Reply by xlg1987** [reply]
我还是没看懂，可以写中文操作手册吗？我完全不懂 ，我就知道会GT

**Reply by rafa0128** [reply]
> 我还是没看懂，可以写中文操作手册吗？我完全不懂 ，我就知道会GT

readme有中文文档，工具也可以切中文，操作基本没有什么复杂操作，就是选好设备和app点击start就开始收集了。

**Comment by WG-itgm** [comment]
我部署好了
&nbsp;


王刚
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年6月5日(星期一) 中午11:02
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [smart-test-ti/SoloX] 这个如何部署到本地服务器上面啊 (Discussion #152)





 
我还是没看懂，可以写中文操作手册吗？我完全不懂 ，我就知道会GT

**Reply by xlg1987** [reply]
我加你qq沟通一下可以吗？

**Comment by WG-itgm** [comment]
微信吧
&nbsp;


王刚
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年6月5日(星期一) 下午2:09
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [smart-test-ti/SoloX] 这个如何部署到本地服务器上面啊 (Discussion #152)





 
我加你qq沟通一下可以吗？

**Comment by WG-itgm** [comment]
lovingyoyo0805




王刚
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "smart-test-ti/SoloX"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年6月5日(星期一) 下午2:09
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [smart-test-ti/SoloX] 这个如何部署到本地服务器上面啊 (Discussion #152)





 
我加你qq沟通一下可以吗？
